web/portal/client/src/entities/Voicemail/Voicemail.tsx

```diff
@@ -59,12 +60,16 @@ const properties: VoicemailProperties = {
     null: _('Unassigned'),
     default: '__null__',
   },
+  relUserIds: {
+    label: _('Users'),
+    type: 'array',
```

Comment:
```suggestion
    type: 'array',
    $ref: '#/definitions/User'
```
This should make ForeignKeyGetter customizations unnecessary

Reply:
Done, but this didn‘t got marked as "outdated" for some reason